android-base/src/main/java/io/github/droidkaigi/confsched2020/MainActivity.kt

```diff
@@ -92,8 +94,21 @@ class MainActivity : DaggerAppCompatActivity() {
         setupNavigation()
         setupStatusBarColors()
 
-        binding.drawerLayout.doOnApplyWindowInsets { _, insets, _ ->
+        binding.drawerLayout.doOnApplyWindowInsets { _, insets, initialState ->
             binding.drawerLayout.setChildInsetsWorkAround(insets)
+            binding.contentContainer.updatePadding(
+                left = insets.systemWindowInsetLeft + initialState.margins.left,
```

Comment:
Thank you!
I think we should use paddings for updating padding. 👀 
```
initialState.**paddings**.left
```

Reply:
🤦‍♂It's a mistake. I changed the wrong codes. :)